roles/olm/files/12-alm-operator.deployment.yaml

```diff
@@ -0,0 +1,48 @@
+##---
+# Source: olm/templates/12-alm-operator.deployment.yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: alm-operator
+  namespace: operator-lifecycle-manager
+  labels:
+    app: alm-operator
+    tectonic-operators.coreos.com/managed-by: tectonic-x-operator
+spec:
+  strategy:
+    type: RollingUpdate
+  replicas: 1
+  selector:
+    matchLabels:
+      app: alm-operator
+  template:
+    metadata:
+      labels:
+        app: alm-operator
+    spec:
+      serviceAccountName: alm-operator-serviceaccount
+      containers:
+        - name: alm-operator
+          command:
+          - /bin/alm
+          image: quay.io/coreos/catalog@sha256:cd7325c8aacdd08d39b639fb70cc978e426ebcc159965ea94aaba07727b15dd8
```

Comment:
It would be nice to have an imagestream for this, so that we could target release-specific images and support detached installs

Reply:
Also, this image seems to be unavailable

Reply:
Re: imagestreams - why wouldn't we just update this definition in openshift-ansible and have people run the playbook?

There was a snag in our build process for ansible that I've just fixed, so I'll push a new version with correct refs.